src/Microsoft.DotNet.HotReload.Utils.Generator.Data/UpdateHandlerInfo.cs

```diff
@@ -0,0 +1,15 @@
+using System.Collections.Immutable;
```

Comment:
this should have the copyright header
```suggestion
// Licensed to the .NET Foundation under one or more agreements.
// The .NET Foundation licenses this file to you under the MIT license.

using System.Collections.Immutable;
```